go/vt/vtctl/cells_aliases.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreedto in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vtctl
+
+import (
+	"flag"
+	"fmt"
+	"strings"
+
+	"golang.org/x/net/context"
+
+	"vitess.io/vitess/go/vt/wrangler"
+
+	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
+)
+
+// This file contains the CellsAliases command group for vtctl.
+
+const cellsAliasesGroupName = "CellsAliases"
+
+func init() {
+	addCommandGroup(cellsAliasesGroupName)
+
+	addCommand(cellsAliasesGroupName, command{
+		"AddCellsAlias",
+		commandAddCellsAlias,
+		"[-cells <cell,cell2...>] <alias>",
+		"Registers a local topology service in a new cell by creating the CellInfo with the provided parameters. The address will be used to connect to the topology service, and we'll put Vitess data starting at the provided root."})
+
+	addCommand(cellsAliasesGroupName, command{
+		"UpdateCellInfo",
```

Comment:
Shouldn't this be `UpdateCellsAlias`? Same for `Delete` below.
Description needs to change also.